Resources/config/services.php

```diff
@@ -79,5 +81,16 @@
             ref('router')
         ])
         ->tag('console.command')
-    ;
+        ->set('sofascore.purgatory.purger.default', DefaultPurger::class);
+
+    if (class_exists('Symfony\Component\HttpKernel\HttpCache\HttpCache')) {
+        $container->services()
+        ->set('sofascore.purgatory.purger.symfony', SymfonyPurger::class)
+            ->args(
+                [
+                    ref('http_cache.store'),
+                    param('sofascore.purgatory.host')
```

Comment:
@zlatkoverk `param` function shortcut is not available under Symfony 4.4. Use the following instead:

```php
'%sofascore.purgatory.host%'
```